app/resources/add_note_base.html.template

```diff
@@ -89,6 +104,14 @@
         <div class="end-multi-columns"></div>
       </div>
 
+      <a href="#" class="status-button" style="text-decoration:none">
```

Comment:
Why do you specify text-decoration:none? If you need it, add a class for it, and specify it in css-default.css instead. Generally, we should not specify "style" attribute in HTML. (Exception may be style="display: none;") It is again to separate the structure and look-and-feel.